lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -2607,6 +2611,9 @@ void glow::fold(Function *F, CompilationMode mode) {
 }
 
 void glow::optimize(Function *F, CompilationContext &cctx) {
```

Comment:
I think it would be useful to log the start/end of the scopes of individual optimization passes (e.g. `DCE`) also. Logging only the scopes for `glow::optimize`, `glow::lower` and `glow::fold` is too coarse in my opinion.

Reply:
I think so too.  Will add in following prs